api/logs/src/main/java/io/opentelemetry/api/logs/LoggerBuilder.java

```diff
@@ -32,6 +34,14 @@ public interface LoggerBuilder {
    */
   LoggerBuilder setInstrumentationVersion(String instrumentationScopeVersion);
 
+  /**
+   * Specify that emitted log records SHOULD NOT automatically include trace context from the {@link
+   * Context#current()}.
+   *
+   * @return this
+   */
+  LoggerBuilder excludeTraceContext();
```

Comment:
👍

Reply:
the name of this seems weird to me... since it's only about the _span_ context, not the trace as a whole. 🤔  Is this something that should be adjusted in the spec?

Reply:
Actually, one thing that may prevent us from doing it is that we consider it to be valid for a Log Record to reference a trace_id but have no associated span_id. You won't encounter it in Otel-generated data but some weird external data sources like this may be possible so we allowed for it in the data model.

Reply:
> I think we have been using Trace Context and Span Context interchangeably a lot.

That's been my experience as well. Here are usage of  [trace context](https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-specification+trace+context&type=code), [tracecontext](https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-specification+tracecontext&type=code), [span context](https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-specification+span+context&type=code) and [spancontext](https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-specification+spancontext&type=code) in the specification.

I don't feel strongly either way about this one.

Reply:
If span context and trace context are used interchangeably, its probably not a big deal for otel java to name this `excludeSpanContext` instead, given that we have a [SpanContext](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/trace/SpanContext.java) interface which would be good to reference in the javadoc.